tests/qos/qos_sai_base.py

```diff
@@ -2590,36 +2590,42 @@ def change_lag_lacp_timer(self, duthosts, get_src_dst_asic_and_duts, tbinfo, nbr
                 dutTestParams["basicParams"]["platform_asic"] == "broadcom-dnx"):
             src_dut = get_src_dst_asic_and_duts['src_dut']
             dst_dut = get_src_dst_asic_and_duts['dst_dut']
-            if src_dut.sonichost.is_multi_asic and dst_dut.sonichost.is_multi_asic:
-                dst_mgfacts = dst_dut.get_extended_minigraph_facts(tbinfo)
-                dst_port_id = dutConfig['testPorts']['dst_port_id']
-                dst_interface = dutConfig['dutInterfaces'][dst_port_id]
-                lag_name = ''
-                for port_ch, port_intf in dst_mgfacts['minigraph_portchannels'].items():
-                    if dst_interface in port_intf['members']:
-                        lag_name = port_ch
+            dst_mgfacts = dst_dut.get_extended_minigraph_facts(tbinfo)
+            dst_interfaces = []
+            dst_interfaces.append(dutConfig['dutInterfaces'][dutConfig['testPorts']['dst_port_id']])
+            dst_interfaces.append(dutConfig['dutInterfaces'][dutConfig['testPorts']['dst_port_2_id']])
+            dst_interfaces.append(dutConfig['dutInterfaces'][dutConfig['testPorts']['dst_port_3_id']])
+            lag_names = []
+            for port_ch, port_intf in dst_mgfacts['minigraph_portchannels'].items():
+                for member in port_intf['members']:
+                    if member in dst_interfaces:
+                        lag_names.append( port_ch )
```

Comment:
Is it possible for any two dst ports to be members of the same port channel? Then it would be better to use a set rather than a list as the lag_names variable type.